code/shared-mobility/shared-mobility-smrt.js

```diff
@@ -158,3 +158,210 @@ $(document).on("knack-scene-render.scene_339", function () {
 $(document).on("knack-scene-render.scene_340", function () {
   disableBreadCrumbsNonAdmin();
 });
+
+
+/******* Associate impoundments with invoice *******/
+
+function getTodaysDate() {
+  var today = new Date()
+  return((today.getMonth() + 1) + "/" + today.getDate() + "/"+ today.getFullYear())
+}
+
+var headers = {
+  "X-Knack-Application-ID": "6669fb3cd43ca60027942eef",
```

Comment:
we'll update this to 5eb46c3fed64e10015c6216e to match the prod app 🔖

Reply:
actually, i just remember there is a way to get this from the Knack object
`Knack.Api.application_id`